src/Agent.Worker/Build/BuildJobExtension.cs

```diff
@@ -40,14 +41,14 @@ public BuildJobExtension()
         }
 
         // 1. use source provide to solve path, if solved result is rooted, return full path.
-        // 2. prefix default path root (build.sourceFolder), if result is rooted, return full path.
-        public void GetRootedPath(IExecutionContext context, string path, out string rootedPath)
+        // 2. prefix default path root (build.sourcesDirectory), if result is rooted, return full path.
+        public string GetRootedPath(IExecutionContext context, string path)
```

Comment:
Removing the out param turned out not to be necessary... long story. I'm planning to go ahead and leave it like this since this is equivalent.